crm_timesheet/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+CRM Timesheet
+=============
+
+This module allows to generate timesheets from leads/opportunities and phone
+calls.
+
+
+Usage
+=====
+
+* In lead/opportunity forms you have Timesheet page.
+* In phone calls tree, you can add an analytic account, and if the call has
+  any duration, a timesheet activity will be created.
```

Comment:
Add: "You can choose how to input time: (1) set time you dedicated to call after finishing it OR (2) create & save call with time set in 00:00 and Stop the timer when call ends, this will set exactly the time you spent"
![02-09-2015 10-22-58](https://cloud.githubusercontent.com/assets/8736623/9626284/bed16ade-515c-11e5-96ad-118570eaf0b1.png)